### PR TITLE
Jihwan/submit error

### PR DESCRIPTION
**Pull request by jhseung**
Man these random commit messages -.-

### CODE REVIEW

File: TCAT/Controllers/ActionOnboardViewController.swift [hunk 1]
```diff
@@ -59,6 +59,37 @@ class ActionOnboardViewController: UIViewController, CLLocationManagerDelegate {
...
+        }
+        image.image = getImage()
+        
+        title.font = UIFont(name: FontNames.SanFrancisco.Bold, size: 28)
+        title.textColor = UIColor.primaryTextColor
+        title.text = getTitle()
+        title.center = view.center
+        title.textAlignment = .center
+        title.snp.makeConstraints { (make) in
+            make.bottom.equalToSuperview().offset(-244)
```
Comment (mattbarker016) [inline TCAT/Controllers/ActionOnboardViewController.swift]: These values are on the larger side @AAAstorga

File: TCAT/InformationViewController.swift [hunk 2]
```diff
@@ -0,0 +1,125 @@
...
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        
+        title = "About Us"
+        
+        view.backgroundColor = UIColor.tableBackgroundColor
+        navigationController?.navigationBar.tintColor = .primaryTextColor
+        
+        view.addSubview(titleLabel)
```
Comment (mattbarker016) [inline TCAT/InformationViewController.swift:32]: adding a subview is the last step. You should only add it once you've formatted it how you like. It's on the order of ms, but I feel like it's good practice

File: TCAT/InformationViewController.swift [hunk 3]
```diff
@@ -0,0 +1,125 @@
...
+            make.height.equalTo(34)
+//            make.bottom.equalTo(view.snp.centerY).offset(-10)
+            make.centerX.equalToSuperview()
+        }
+        
+        sendFeedbackButton.addTarget(self, action: #selector(openBugReportForm), for: .touchUpInside)
+        sendFeedbackButton.setTitle("Send Feedback", for: .normal)
+        sendFeedbackButton.setTitleColor(UIColor.tcatBlueColor, for: .normal)
+        sendFeedbackButton.titleLabel?.font = UIFont(name: FontNames.SanFrancisco.Regular, size: 14)
+        sendFeedbackButton.backgroundColor = UIColor.white
```
Comment (mattbarker016) [inline TCAT/InformationViewController.swift]: You can use just `.white`

**Review by mattbarker016 — COMMENTED** [review]
I made a CustomNavigationController, so I may use it where you instantiate a UINavigationController. Merge in for now after some of the fixes, and I'll build on my device and go from there

File: TCAT/Controllers/HomeViewController.swift [hunk 4]
```diff
@@ -410,6 +425,21 @@ class HomeViewController: UIViewController, UITableViewDelegate, UITableViewData
...
+    
+    /* Open information screen */
+    @objc func openInformationScreen() {
+        let informationViewController = InformationViewController()
+        let navigationVC = UINavigationController(rootViewController: informationViewController)
+        self.present(navigationVC, animated: true, completion: nil)
+        
+//        let transition: CATransition = CATransition()
+//        transition.duration = 0.5
+//        transition.timingFunction = CAMediaTimingFunction(name: kCAMediaTimingFunctionEaseInEaseOut)
```
Comment (mattbarker016) [inline TCAT/Controllers/HomeViewController.swift]: Are you planning to use this code?

File: TCAT/InformationViewController.swift [hunk 5]
```diff
@@ -0,0 +1,125 @@
...
+            make.top.equalTo(sendFeedbackButton.snp.bottom).offset(-0.5)
+        }
+    }
+    
+    @objc func backButtonClicked() {
+        self.dismiss(animated: true, completion: nil)
+    }
+    
+    @objc func openBugReportForm() {
+        let betaFormURL = "https://goo.gl/forms/u2shinl8ddNyFuZ23"
```
Comment (mattbarker016) [inline TCAT/InformationViewController.swift:115]: You have the link in two places. Maybe it should be a constant somewhere else so we can update it just in one place

File: fastlane/report.xml [hunk 6]
```diff
@@ -0,0 +1,48 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (mattbarker016) [inline fastlane/report.xml]: Hmm, the .gitignore should prevent this. Do you maybe not have the latest from master?

File: TCAT/Controllers/ActionOnboardViewController.swift (hunk 1 above)
Comment (jhseung) [reply TCAT/Controllers/ActionOnboardViewController.swift]: Imma have to fix this–I think I didn't merge properly with master, this is def not right.

File: TCAT/InformationViewController.swift (hunk 5 above)
Comment (jhseung) [reply TCAT/InformationViewController.swift:115]: Not sure what you mean by this

File: TCAT/InformationViewController.swift (hunk 2 above)
Comment (jhseung) [reply TCAT/InformationViewController.swift:32]: But to use constraints, I have to add them to the view before setting them?

File: TCAT/InformationViewController.swift (hunk 5 above)
Comment (mattbarker016) [reply TCAT/InformationViewController.swift:115]: Oh my bad, I thought for some reason you had the URL in two places

File: TCAT/InformationViewController.swift (hunk 2 above)
Comment (mattbarker016) [reply TCAT/InformationViewController.swift:32]: Oh right, using snapkit and all

**Pull request merged**